Mathlib/RingTheory/DedekindDomain/Ideal.lean

```diff
@@ -1538,4 +1539,41 @@ theorem multiplicity_normalizedFactorsEquivSpanNormalizedFactors_symm_eq_multipl
     multiplicity_normalizedFactorsEquivSpanNormalizedFactors_eq_multiplicity hr ha]
 #align multiplicity_normalized_factors_equiv_span_normalized_factors_symm_eq_multiplicity multiplicity_normalizedFactorsEquivSpanNormalizedFactors_symm_eq_multiplicity
 
+/-- The bijection `normalizedFactorsEquivSpanNormalizedFactors.symm` between the set of prime
```

Comment:
`normalizedFactorsEquivSpanNormalizedFactors.symm` does not appear in the statement of the theorem...